### PR TITLE
Created Mentor App Button and Applied Button Hovering

### LINKED ISSUE #157: Create Button to Apply
**DUE DATE FOR HACKERS: 3/6**
**DUE DATE FOR MENTORS: 3/16**

- Create button below VH title to apply.
- Same button componentused for both hacker + mentor application.

### PR DESCRIPTION
**Pull request by nathantoannguyen**
## Summary
Created mentor app button, which will redirect to [mentor applications](https://airtable.com/shr5Y9x5ocJaJL52j), and reapplied button hovering when hovering over the buttons so that it'll pop out a bit (or idk how to describe it)

## Test Plan
image

## Issue(s)
Closes #157 and #172

## Future Followup

### CODE REVIEW

File: src/app/views/home/Home.scss [hunk 1]
```diff
@@ -148,7 +148,31 @@ $laptop: "./../../../assets/images/new-laptop.png";
...
+            text-decoration: none;
+        }
+      }
+
+      #apply-hacker-btn:hover {
+        transition: all .2s ease;
+        transform: scale(1.08);
+      }
+
+      #apply-mentor-btn:hover {
```
Comment (ChaseC99) [inline src/app/views/home/Home.scss:175]: These buttons have the same styling. This should be a class that is re-used by both, not an id.

**Review by ChaseC99 — CHANGES_REQUESTED** [review]

**Comment by nathantoannguyen** [comment]
If you could look over my latest changes whenever you can @ChaseC99 (hopefully for the last time for the affected issues), that'll be greatly appreciated 🙏

File: src/app/views/home/Home.js [hunk 2]
```diff
@@ -120,6 +120,21 @@ export default class Home extends React.Component {
...
+   */
+
+  ApplyButton(URL, text) {
+    return (
+      <Button className="apply-btn"
+        href={URL} target="_blank" rel="noopener noreferrer">
+          {text}
+      </Button>
+      )
+  }
```
Comment (ChaseC99) [inline src/app/views/home/Home.js:137]: This should be a function in a separate file that gets exported. Put it in the `/components` folder.

File: src/app/views/home/Home.js [hunk 3]
```diff
@@ -158,11 +173,8 @@ export default class Home extends React.Component {
                   <img id="venushacks-title" src={vh_title} alt="VenusHacks Title Logo"/>
                   <h4 id="date">May 20 - 22, 2022</h4>
                   <p id="tagline">UC Irvine's largest women-centric hackathon</p>
-                    <Button id="apply-btn">
-                      <a href="https://airtable.com/shr1Etr4Typ3FEqBc" target="_blank" rel="noopener noreferrer">
-                        Apply as a Hacker
-                      </a>
-                    </Button>
+                      {this.ApplyButton("https://airtable.com/shr1Etr4Typ3FEqBc", "Apply as a Hacker")}
+                      {this.ApplyButton("https://airtable.com/shr5Y9x5ocJaJL52j", "Apply as a Mentor")}
```
Comment (ChaseC99) [inline src/app/views/home/Home.js:177]: When we import it as a component, we no longer need `this` and we can be explicit with the props.
```suggestion
                      <ApplyButton url="https://airtable.com/shr1Etr4Typ3FEqBc" text="Apply as a Hacker" />
                      <ApplyButton url="https://airtable.com/shr5Y9x5ocJaJL52j" text="Apply as a Mentor" />
```

File: src/app/views/home/Home.js [hunk 4]
```diff
@@ -120,6 +120,21 @@ export default class Home extends React.Component {
...
     )
   }
+  /**
+   * 
+   * @param {*} URL url of respective application
+   * @param {*} text apply as {text}
+   * @returns button of respective application
+   */
+
+  ApplyButton(URL, text) {
```
Comment (ChaseC99) [inline src/app/views/home/Home.js:130]: make url lowercase
```suggestion
  ApplyButton(url, text) {
```

**Review by ChaseC99 — CHANGES_REQUESTED** [review]
Sorry for all the revisions... Just one more quick round of changes and it should be good to go.

**Comment by ChaseC99** [comment]
I apologize for the delay, I didn't realize you were going to d0 #172 in this PR too. I thought we'd merge, deploy, and then refactor this later.

**Comment by ChaseC99** [comment]
I hope you don't mind but since this time sensitive, I went ahead and pushed my own commit with those changes. Look good to you @nathantoannguyen?

**Comment by nathantoannguyen** [comment]
Yeah, no worries, it looks good to me based on what you said and did. Thank you so much Chase, I'm obviously still unfamiliar with React so it'll make sense that you would request changes from how I implemented them, but I'm learning a lot from your help! I'll approve the changes @ChaseC99

**Review by ChaseC99 — APPROVED** [review]

**Review by nathantoannguyen — COMMENTED** [review]
Makes sense to utilize the components folder 👍 I understand components and props better now

**Pull request merged**